configs/config/debugging/pretrain/supervised/supervised_1gpu_vision_transformer_debug_integration_cutmix.yaml

```diff
@@ -35,7 +35,6 @@ config:
       }
     TEST:
       DATA_SOURCES: [disk_folder]
-      #      DATA_PATHS: ["<path to test folder>"]
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.